clacheck/clacheck.py

```diff
@@ -55,14 +57,14 @@ def update_status(pr, state, description):
     conn.getresponse().read()
     host,url = url_split(pr['issue_url'])
     if state == SUCCESS:
-        url = url + '/labels/need-cla'
+        url = url + '/labels/'+ urllib.quote(CLA_LABEL)
```

Comment:
Same here, space before the `+` too

Reply:
Ups, didn't read far enough. Amended and force-pushed my fixup, see ac4e41a1386419e044448b019e83e575133523df.